### PR TITLE
[16.0][MIG] website_sale_product_detail_attribute_image: migration to 16.0

### PR DESCRIPTION
**Pull request by Segui-S73**
Standard migration from [15.0](https://github.com/OCA/e-commerce/tree/15.0/website_sale_product_detail_attribute_image)

### CODE REVIEW

File: website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js [hunk 1]
```diff
@@ -22,7 +27,7 @@ odoo.define("website_sale_product_detail_attribute_image.tour", function (requir
         {
             url: "/shop",
             test: true,
-            wait_for: base.ready(),
+            wait_for: ready,
```
Comment (chienandalu) [inline website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js]: Is it needed at all?

**Review by chienandalu — APPROVED** [review]
LGTM. Tiny comment

**Comment by pedrobaeza** [comment]
@Segui-S73 can you answer the comment? And one thing, isn't this something included already in core?

File: website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js (hunk 1 above)
Comment (Segui-S73) [reply website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js]: It is not necessary, in 15.0 it was configured with the `wait_for` parameter and I migrated it as a precaution, I am going to delete it.
Sorry for the delay.

Comment (Segui-S73) [reply website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js]: Done

**Comment by pedrobaeza** [comment]
/ocabot migration website_sale_product_detail_attribute_image
/ocabot merge nobump

**Pull request merged**